Inconsistent golang versions in different Dockerfiles

It seems we are using inconsistent golang versions in [Dockerfile.builder-musllinux](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile.builder-musllinux), [Dockerfile.builder-manylinux](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile.builder-manylinux) and [Dockerfile](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile). 

Trying to build the development container using the readme file causes the following error. 
```
#6 1.656 src/github.com/seccomp/libseccomp-golang/seccomp_internal.go:698: cannot use req.data.arch (type C.__u32) as type C.uint32_t in argument to archFromNative
------
executor failed running [/bin/sh -c go get github.com/seccomp/libseccomp-golang &&     go get github.com/fatih/color &&     go get github.com/gobwas/glob &&     go get gopkg.in/yaml.v2]: exit code: 2
make: *** [prepare-dev] Error 1
```

Since [Dockerfile.builder-musllinux](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile.builder-musllinux) and [Dockerfile.builder-manylinux](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile.builder-manylinux) uses golang 1.11 version and [Dockerfile](https://github.com/lablup/backend.ai-jail/blob/main/Dockerfile) uses golang 1.8 version, I think the golang version for the `Dockerfile` should be updated to 1.11 version.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.8-alpine
+FROM golang:1.11-alpine
 # This container is for daily development.
 
 RUN apk add --no-cache build-base git libseccomp-dev linux-headers

